defaults/main.yml

```diff
@@ -4,9 +4,11 @@ openwisp2_python: python3
 ansible_python_interpreter: /usr/bin/python3
 openwisp2_network_topology: false
 openwisp2_firmware_upgrader: false
+openwisp2_monitoring: true
```

Comment:
The default should be `false`

Reply:
true is good, most users need this, only advanced users which have a pre-existing monitoring solution which they integrate need to turn this off